Src/HALALMock/Services/SharedMemory/SharedMemory.cpp

```diff
@@ -1,8 +1,34 @@
 
 #include "HALALMock/Services/SharedMemory/SharedMemory.hpp"
-
+//includes to create the shared Memory in Posix
+#include <fcntl.h>
+#include <sys/shm.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <unistd.h>
 void SharedMemory::start() {
-    // TODO: create / open shared memory
+    //Create GPIO_Memory
+	const char* 
+	int shm_gpio_fd;
+	//create shared memory object
+	shm_gpio_fd = shm_open(gpio_memory_name, O_CREAT | O_RDWR,0666);
```

Comment:
'0666' mode allows to everybody to read and write on this file descriptor, it wouldn't better to use 0660? It would allow to propietary and group to read and write, but rest of users couldn't do anything
```suggestion
	shm_gpio_fd = shm_open(gpio_memory_name, O_CREAT | O_RDWR,0660);
```

Reply:
Yes, it's be a good Idea, for safety reasons.